src/main/java/org/gridsuite/actions/server/ContingencyListService.java

```diff
@@ -142,9 +142,25 @@ private List<Contingency> getPowsyblContingencies(PersistentContingencyList cont
         return powsyblContingencyList == null ? Collections.emptyList() : powsyblContingencyList.getContingencies(network);
     }
 
-    List<ContingencyInfos> exportContingencyList(UUID id, UUID networkUuid, String variantId) {
+    @Transactional(readOnly = true)
+    public Integer getContingencyCount(List<UUID> ids, UUID networkUuid, String variantId) {
+        Network network = getNetworkFromUuid(networkUuid, variantId);
+        return ids.stream()
+            .map(uuid -> {
+                List<ContingencyInfos> contingencyList = exportContingencyList(uuid, network);
```

Comment:
return exportContingencyList(uuid, network).size() ?
and return List.of() from exportContingencyList

Reply:
I did not want to change the existing function, was not sure about the impact.